pkg/util/utils.go

```diff
@@ -111,10 +111,20 @@ func (m MultiError) ToError() error {
 // EscapeStringForSSHUse will parse string and replace
 // shell special characters with escape
 func EscapeStringForSSHUse(s string) string {
-	r := strings.NewReplacer("$", "\\$",
+	r := strings.NewReplacer(
+		"$", "\\$",
 		`"`, `\"`,
 		"`", "\\`",
-		`\`, `\\`)
+		`\`, `\\`,
+		`(`, `\(`,
+		`)`, `\)`,
+		`{`, `\{`,
```

Comment:
-1, this is not about escaping special shell characters. This is a specific escape for using a string as part of an ssh command. There is only a limited amount of characters which needs to be escaped there.

On top of this, even in a shell, {, }, [, ] would not have a special meaning.

I think we can actually close this pull request and resolve also the issue. The solution is to put the password into single quotes (instead of double) to prevent substitution within the shell.